microsetta_interface/templates/sample_results.jinja2

```diff
@@ -0,0 +1,104 @@
+{% extends "sitebase.jinja2" %}
+{% set page_title = "Sample Results" %}
+{% set show_breadcrumbs = True %}
+{% set show_logout = True %}
+
+{% block head %}
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.css" />
```

Comment:
should all these sources come from `static/`?

Reply:
What are the pros/cons of that vs. using a CDN?

Reply:
Note sure TBH. @dhakim87?

Reply:
Updated in https://github.com/biocore/microsetta-interface/pull/11/commits/07ad4628e717c96ae820900a0f2783a6ad09f844